src/app/components/team/SmoochBot/SmoochBotMainMenuOption.js

```diff
@@ -30,7 +30,7 @@ const SmoochBotMainMenuOption = ({
   const handleSave = () => {
     setSubmitted(true);
     if (text && value) {
-      onSave(text, description, value, currentKeywords);
+      onSave(text, description, value, id, currentKeywords);
```

Comment:
I need help here as I expect `id` to be undefined. Where would the current value be stored

Reply:
@danielevalverde did you address this in this branch?